deploy/charts/mixer/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: mixer
+description: A Helm chart for Data Commons Mixer API server.
```

Comment:
Ditto the comment about chart being confusing file name -- HelmChart.yaml perhaps?

Reply:
Unfortunately this file has to be Chart.yaml. Added a comment in the file.